docs/server/upgrading.rst

```diff
@@ -3,170 +3,222 @@
 Upgrading
```

Comment:
We don't make any mention of taking the server offline when upgrading.  It might be worth mentioning it so that people aren't working on the site when we upgrade.